locale/fi/LC_MESSAGES/django.po

```diff
@@ -3696,3 +3696,6 @@ msgstr "Unohda automaattinen uudelleenohjaus."
 #: userprofile/views.py:80
 msgid "No privacy notice. Please notify administration!"
 msgstr "Ei tietosuojailmoitusta. Ota yhteys ylläpitäjään!"
+
+#~ msgid "View course"
```

Comment:
Could you run compilemssages with argument `--no-obsolete`, because these don't need to be kept in the file (i.e. git stores old valuesi in the history).

Reply:
All right, will do

Reply:
There shouldn't be any more those extra values left in django.po